recipes/newmat/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "11.0.1":
+    url: "http://www.robertnz.net/ftp/newmat11.tar.gz"
+    sha256: "daf313a4db0db14dfcce236ad3cdc514d449e62ddd4f1b07cc31179b0787db6f"
+patches:
+  "11.0.1":
+    - patch_file: "patches/11.0.1/newmat6.cpp.patch"
```

Comment:
There is no reference about these patches. Please, follow  https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patches-fields and in case they really required, please, send them to the author too, so we don't need to be the maintainer.

Reply:
Fixed in 6fadd505c804c514f6abf133cd5a78b847d37fef

Reply:
Would you prefer just "compilation fix for c++11 compatible compilers"